docs/configuration.asciidoc

```diff
@@ -1066,7 +1067,14 @@ module.exports = {
 * *Default:* `true`
 * *Env:* `ELASTIC_APM_BREAKDOWN_METRICS`
 
-Used to disable reporting of breakdown metrics which records self time spent in each unique type of span.
+Set `breakdownMetrics: false` to disable reporting of breakdown metrics. Note
+that if `metricsInterval: 0`, then breakdown metrics will not be reported.
+
+Breakdown metrics (<<metrics-span.self_time.sum, `span.self_time.*`>>) record
+the self-time spent in each unique type of span. This data drives the
+{kibana-ref}/service-overview.html#service-span-duration[Time spent by span type]
```

Comment:
@bmorelli25 Can I make this `{kibana-ref}/service-overview.html#service-span-duration` reference? When I do a local build of the docs to check it links to the *7.10* version of Kibana docs: https://www.elastic.co/guide/en/kibana/7.10/service-overview.html#service-span-duration  (which is a 404).

Reply:
When was the last time you updated your local copy of the https://github.com/elastic/docs repo? My guess is not since 7.10 😁. Links to the stack are governed by that [`current.asciidoc`](https://github.com/elastic/docs/blob/master/shared/versions/stack/current.asciidoc) file I showed you earlier. I'm guessing yours is set to 7.10. A quick pull from upstream/master should fix things right up!

Reply:
```
% cat ../docs/shared/versions/stack/current.asciidoc
include::7.10.asciidoc[]
```

There you go. :) Thanks!